x/launch/keeper/chain_test.go

```diff
@@ -57,6 +58,7 @@ func TestKeeper_CreateNewChain(t *testing.T) {
 			sourceHash:     sample.String(20),
 			genesisURL:     "",
 			hasCampaign:    false,
+			metadata:       []byte(sample.String(20)),
```

Comment:
From my comment above, I think it actually makes sense to implement a `sample.Metadata` for better maintainability